lib/src/core/utils/bloc_utils.dart

```diff
@@ -0,0 +1,68 @@
+import 'state.dart';
+import 'ui_resource_state.dart';
+
+import '../../domain/entities/genre.dart';
+import '../../domain/entities/movies_and_genres_response.dart';
+
+class BlocUtils {
+  static List<Genre> getGenres(List<int> ids, List<Genre> genres) {
+    List<Genre> selectedGenres = [];
+    selectedGenres = genres
+        .where(
+          (genre) => ids.contains(genre.id),
+        )
+        .toList();
+    return selectedGenres;
+  }
+
+  static UiResourceState mapToMovieAndGenresResponse(
```

Comment:
Why this is called on a static method on a util class?

Reply:
You are right, i changed them to global methods